core/src/main/java/be/isach/ultracosmetics/cosmetics/particleeffects/ParticleEffectHalo.java

```diff
@@ -0,0 +1,32 @@
+package be.isach.ultracosmetics.cosmetics.particleeffects;
+
+import be.isach.ultracosmetics.UltraCosmetics;
+import be.isach.ultracosmetics.cosmetics.type.ParticleEffectType;
+import be.isach.ultracosmetics.player.UltraPlayer;
+import be.isach.ultracosmetics.util.MathUtils;
+
+import org.bukkit.Location;
+
+public class ParticleEffectHalo extends ParticleEffect {
```

Comment:
Should halos do an alternative effect when moving? They look a little strange to me when you're moving:
![2022-12-31_11 33 43](https://user-images.githubusercontent.com/17661376/210154063-7219c376-7025-4725-9da0-0aee4dab9279.png)

Reply:
Fixed. All halos have the alternative effect now.